### DISCUSSION TITLE
What is the default font family which UI kitten eva theme uses?

### DISCUSSION THREAD

**Discussion by rpsutton** [akveo/react-native-ui-kitten#1373] [Q&A]
I really like the font and I want to write my web app with the matching font family.

**Comment by artyorsh** [comment]
It is [Open Sans](https://fonts.google.com/specimen/Open+Sans)

**Reply by rpsutton** [reply]
Thank you!

**Comment by hellygeorge** [comment]
You call createTheme with an object that has the typography Mmcgbl!
